examples/log-api/log_api.bal

```diff
@@ -4,13 +4,19 @@ public function main() {
     error e = error("something went wrong!");
 
     // There are two log levels (INFO, ERROR) and there is no user configuration to control which log level to print.
-    // Users can pass any number of key/value pairs which needs to be displayed in the log message.
     log:print("info log");
-    log:print("info log", id = 845315, name = "foo");
+    log:printError("error log");
+
+    // Users can pass any number of key/value pairs which needs to be displayed in the log message.
+    // These can be of anydata type including int, string and boolean.
```

Comment:
```suggestion
    // These can be of any data type including int, string, and boolean.
```

Reply:
IINM isn't "any data" type different to `anydata` type in Ballerina.

Reply:
Shall we add backtick(`) for anydata

Reply:
Updated in 68a609c

Reply:
+1